modules/log/file.go

```diff
@@ -243,6 +243,11 @@ func (log *FileLogger) deleteOldLog() {
 	})
 }
 
+// Content returns the content accumulated in the content provider
+func (log *FileLogger) Content() (string, error) {
+	return log.Filename, nil
```

Comment:
In this function it seems like you rather need to return the contents of the file?

Reply:
You are correct. Fixed.